openrtb2/bid.go

```diff
@@ -269,6 +303,39 @@ type Bid struct {
 	//   wait between the auction and the actual impression.
 	Exp int64 `json:"exp,omitempty"`
 
+	// Attribute:
+	//   dur
+	// Type:
+	//   integer
+	// Description:
+	//   Duration of the video or audio creative in seconds.
+	Dur int64 `json:"dur,omitempty"`
+
+	// Attribute:
+	//   mtype
+	// Type:
+	//   integer
+	// Description:
+	//   Type of the creative markup so that it can properly be
+	//   associated with the right sub-object of the BidRequest.Imp.
+	//   Values:
+	//   1 = Banner
+	//   2 = Video
+	//   3 = Audio
+	//   4 = Native
+	MType int8 `json:"mtype,omitempty"`
```

Comment:
TODO: maybe import adcom1 here or extract enum here

Reply:
There is no list defined in the adcom spec. It's kind of an ad-hoc enum. I'm good if you'd like to extract it.

Reply:
Extracted as MarkupType ^^